src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,99 @@
-export const Pagination = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  return (
+    <>
+      <ul className="pagination">
+        <li className={classNames(
+          'page-item',
+          {
+            disabled: currentPage === 0,
+          },
+        )}
+        >
+          <a
+            data-cy="prevLink"
+            className="page-link"
+            href="#prev"
+            aria-disabled={currentPage === 0}
+            onClick={() => {
+              if (currentPage !== 0) {
+                onPageChange(currentPage - 1);
+              }
+            }}
+          >
+            «
+          </a>
+        </li>
+        {Array(Math.ceil(total / perPage))
+          .fill(null)
+          .map((_, idx) => (
```

Comment:
this logic here looks a bit complicated, its better to move it into variable and use here like a `someArr.map()`

Reply:
yep, move it to variable or to some function